codegen/syntax/AliasDeclaration.hack

```diff
@@ -1,11 +1,13 @@
 /**
  * This file is generated. Do not modify it manually!
  *
- * @generated SignedSource<<51b4ce4fe28d6124a26843b29b1f73f7>>
+ * @generated SignedSource<<d4fc81d224048c5ec9315137fe137818>>
  */
 namespace Facebook\HHAST;
 use namespace Facebook\TypeAssert;
 use namespace HH\Lib\Dict;
+/* HHAST_IGNORE_ALL[5607] 5607 is ignored because of false positives when comparing a generic to a typed value */
+/* HHAST_IGNORE_ALL[5624] HHAST_IGNORE_ALL[5639] 5624 and 5639 are ignored because they insist on using co(tra)variant generics. Could this break external consumers? */
```

Comment:
This linter might also be good for the default blacklist - there's an ongoing discussion about removing it entirely at https://fb.workplace.com/groups/hackforhiphop/posts/7429940763721141

In short: It's accurate, but it's highly opinionated, and there's not consensus on the behavior it encourages being a good thing.

This isn't the same as the other codes we'd want to always-ignore, but "pretend it doesn't exist" may be the right thing to do for "may be deleted soon". Perhaps making it off-by-default but explicitly-includable may be better.